### ISSUE TITLE
MAS 1.4.3: Elements must have sufficient color contrast (.btn-info)

### ISSUE THREAD

**Issue by SaisreeDandamudi** [terrajobst/apisof.net#19]
**GitHubTags:**
#A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_api-catalog; #MAS1.4.3; #C+E-H; #api-catalog; #Benchmark; #A11ySev2;

#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Home - .NET API Catalog](https://apisof.net/)
#### Element path
.btn-info
#### Snippet
`<a class="btn btn-info btn-lg search-link" href="javascript:void(0)">Search</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.09 (foreground color: #ffffff, background color: #5bc0de, font size: 13.5pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 76.0.3809.100
This accessibility issue was found using Accessibility Insights for Web 2.8.2 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
![Colour Contrst search button](https://user-images.githubusercontent.com/51435580/63583607-fa916e00-c5b8-11e9-948b-f3460b392cd8.png)

**Comment by terrajobst** [comment]
The web site no longer exists.
